libpod/healthcheck.go

```diff
@@ -199,3 +239,75 @@ func (c *Container) GetHealthCheckLog() (inspect.HealthCheckResults, error) {
 	}
 	return healthCheck, nil
 }
+
+// Creates systemd timers for healthchecks of a container
+func (c *Container) CreateTimer() error {
+	if c.disableHealthCheckSystemd() {
+		return nil
+	}
+	podman, err := os.Executable()
+	if err != nil {
+		return errors.Wrapf(err, "failed to get path for podman for a health check timer")
+	}
+
+	var cmd = []string{"--unit", fmt.Sprintf("%s", c.ID()), fmt.Sprintf("--on-active=%s", c.HealthCheckConfig().Interval.String()), podman, "healthcheck", "run", c.ID()}
+
+	_, err = dbus.NewSystemdConnection()
```

Comment:
We probably want to do a `defer close()` on the systemd connection

Reply:
ill just close it proper